packages/rlc-common/src/helpers/schemaHelpers.ts

```diff
@@ -3,9 +3,14 @@
 
 import { Schema } from "../interfaces.js";
 
-export function isDictionarySchema(schema: Schema) {
+export function isDictionarySchema(
+  schema: Schema,
+  filterEmpty: boolean = false
```

Comment:
minor comment: I usually prefer to add an optional options bag to hold the single filterEmpty option as it helps with readability

for example

```ts
export interface IsDictionaryOptions {
   filterEmpty?: boolean;
}

export function isDictionarySchema(
  schema: Schema,
  options?: IsModelOptions = {filterEmpty: false}
)
``` 

This produces a call like this

```ts
isDictionary(scheman, {filterEmpty: true});

//vs
isDictionary(scheman, true);
```

It helps when reading the code later on, you don't have to go to the function definition to see what the boolean value means

Reply:
Thanks for the suggestion, I updated.